contributing_to_docs/doc_guidelines.adoc

```diff
@@ -268,6 +271,84 @@ another topic, see the following usage:
 
 http://asciidoctor.org/docs/user-manual/#include-multiple
 
+== Writing concepts
+Concepts should support the things that users want to do and should not include
```

Comment:
@aheslin I can't find the doc you showed me the other day (and Ashley seems to have put most of it in this topic), but was there an example for a concept topic in there? Further to what Alex says above, an example could help define the idea.